### PR TITLE
Dynamic install path

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

**Comment by MeirShpilraien** [comment]
@itamarhaber the PR also contains more configuration documentation changes so if you can review.

File: docs/configuration.md [hunk 1]
```diff
@@ -118,3 +118,93 @@ _Default Value_
...
+## PythonInstallationDir
+The **PythonInstallationDir** configuration option specifies the path for RedisGears' Python dependencies.
+
+_Expected Value_
+
+String
+
+_Default Value_
+
+?
```
Comment (itamarhaber) [inline docs/configuration.md:176]: @MeirShpilraien this needs to be defined

Comment (MeirShpilraien) [reply docs/configuration.md]: Sets to : /var/opt/redislabs/modules/rg

**Comment by MeirShpilraien** [comment]
- [ ] Handle multi/exec and lua

**Pull request merged**